src/common/Interfaces.ts

```diff
@@ -37,10 +37,12 @@ export abstract class AnimatedText extends AnimatedComponent<Types.AnimatedTextP
 export abstract class AnimatedTextInput extends AnimatedComponent<Types.AnimatedTextInputProps, {}> {
 }
 
-export abstract class AnimatedView extends AnimatedComponent<Types.AnimatedViewProps, {}> {
-    abstract focus(): void;
+export abstract class AnimatedView extends AnimatedComponent<Types.AnimatedViewProps, {}> implements FocusableComponent {
     abstract setFocusRestricted(restricted: boolean): void;
     abstract setFocusLimited(limited: boolean): void;
+    abstract focus(): void;
+    abstract realFocus(): void;
```

Comment:
I don't think "realFocus" is a good name. All methods should start with a verb. And "real" is ambiguous in this context. Perhaps we should call this method "focus", and the other method (the one currently called "focus") should be called "requestFocus". This will make it clear that "focus" takes place immediately, the same as "blur". And "requestFocus" is something that is deferred and arbitrated.